.circleci/config.yml

```diff
@@ -61,31 +54,75 @@ jobs:
             # we will always have drift
             # terraform plan -detailed-exitcode
 
-      - run:
+      - run: &terraform_destroy
```

Comment:
FWIW instead of anchors, you can now share "commands": https://github.com/dominodatalab/terraform-aws-eks/pull/21